react-example/src/index.tsx

```diff
@@ -43,7 +43,7 @@ const HomeLink = styled(Link)`
     ({ email }) => {
       return axios
         .post(
-          'http://localhost:5000/generate',
+          process.env.JWT_GENERATOR || '',
```

Comment:
would it be best to at least leave the default as `http://localhost:5000/generate` so the api hits something?